.github/workflows/create-release.yml

```diff
@@ -190,6 +190,7 @@ jobs:
         IMAGE_ORG="quay.io/konveyor"
         VERSION="${{ inputs.version }}"
         CSV="./bundle/manifests/konveyor-operator.clusterserviceversion.yaml"
+        export PATH=$PATH:./bin
```

Comment:
Not sure about relative path in PATH, but worth trying, LGTM.

Reply:
I tested quickly and it should work.